README.md

```diff
@@ -1,4 +1,4 @@
-<p align="center">
+<README.mdp align="center">
```

Comment:
```suggestion
<p align="center">
```

Reply:
yep, I've fixed this now locally. need to push